demos/browser/webpack.config.js

```diff
@@ -18,6 +19,9 @@ module.exports = env => {
         inject: 'head',
       }),
       new HtmlWebpackInlineSourcePlugin(),
+      new webpack.EnvironmentPlugin({
+        IS_LOCAL: process.env.npm_config_is_local === 'true' ? 'true' : 'false'
```

Comment:
Does that mean we have to specify another argument --is_local=true when running npm run start?

Reply:
`start:fast` in `demos/browser/package.json` specifies `--is_local=true` so you don't need to pass any option.